src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,80 @@
+package duke.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.FindException;
+import duke.task.Task;
+
+public class FindCommand extends Command {
```

Comment:
Should the class name be a noun instead (i.e. CommandFinder)?

Reply:
Hi, what I actually meant is "Find" Command. I didn't know it sounds ambiguous. Thank you for pointing it out! Will rename it 👍